documentation/modules/rn-2.2.adoc

```diff
@@ -94,3 +94,37 @@ The error status message for a VM with no operating system on the *Migration pla
 .Network, storage, and VM referenced by name in the Plan CR are not displayed in the web console.
 
 If a Plan CR references storage, network, or VMs by name instead of by ID, the resources do not appear in the {project-short} web console. The migration plan cannot be edited or duplicated. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1986020[*BZ#1986020*])
+
+.Restart of the CatalogSource pod is required to upgrade MTV 2.1.0 to MTV 2.2.0.
+
+To restart the CatalogSource pod, do the following:
+
+.  In the *Subscriptions* tab of the {operator-name} in the {ocp-short} web console, take note of the name in the *CatalogSource* field.
+.  In the CLI, run the following command:
++
+[source,terminal]
+----
+  $ oc get pod -n openshift-marketplace
+----
+Output: A list of pods
+
+.  Find a pod with a name that begins with the CatalogSource name.
+.  Copy the full name of the pod.
+.  Run the following command:
++
+[source,terminal]
+----
+  $ oc delete pod -n openshift-marketplace <pod_name>
+----
+Output:
+
++
+[source,terminal]
+----
+  pod "<pod_name>" deleted
+----
+A new CatalogSource pod is created and started automatically.
+In the *Subscriptions* tab of the {operator-name} in the {ocp-short} web console, *Upgrade status* changes to `Upgrade available`.
+
+You can now proceed with the upgrade.
+(link:https://bugzilla.redhat.com/show_bug.cgi?id=2031709[*BZ#2031709*])
```

Comment:
Put this closer to the beginning of the section, near a sentence that explains what this bug is and why the pod has to be deleted.

Reply:
Also, this link is to the docs bug. 

@mguetta1 Is there an engineering bug that we can link to instead?

Reply:
@apinnick No I will open one

Reply:
@apinnick Actually it is not considered as a bug that should be fixed, just as an extra step.
And this is the reason why there is no technical bug for this

Reply:
@mguetta1 If this is not going to be fixed, will it apply to future upgrades?

Reply:
@apinnick I don't know. If I remember correctly even Dev had no answer for why this step is required now
And it was agreed to add it as an extra mandatory step in MTV doc

Reply:
@sbeskin-redhat 
This procedure belongs to the upgrade section, not Known Issues.

Reply:
@sbeskin-redhat 
Why did you resolve this? You have not made the change I requested in my last comment.

Reply:
@apinnick  
You mean the upgrade procedure in the MTV user guide, not in the RN?